tests/test_sbom.py

```diff
@@ -1,3 +1,6 @@
+import json
+import unittest.mock
+
 import pytest
 import random
 import hashlib
```

Comment:
Generally stdlib imports should come first, followed by third party packages, and finally local modules (see https://peps.python.org/pep-0008/#imports):
```suggestion
import json
import random
import hashlib
import unittest.mock

import pytest

```

Reply:
Thanks @ezio-melotti!